fronts-client/src/components/feed/RecipeSearchContainer.tsx

```diff
@@ -84,20 +95,37 @@ export const RecipeSearchContainer = ({ rightHandContainer }: Props) => {
 
 	const [page, setPage] = useState(1);
 
-	/*const debouncedRunSearch = debounce(() => runSearch(page), 750); TODO need to check if needed for chef-search? if yes then how to improve implementing it*/
-
 	useEffect(() => {
 		const dbf = debounce(() => runSearch(page), 750);
 		dbf();
 		return () => dbf.cancel();
-	}, [selectedOption, searchText, page]);
+	}, [selectedOption, searchText, page, dateField, orderingForce]);
 
 	const chefsPagination: IPagination | null = useSelector((state: State) =>
 		chefSelectors.selectPagination(state),
 	);
 
 	const hasPages = (chefsPagination?.totalPages ?? 0) > 1;
 
+	const getUpdateConfig = () => {
+		switch (orderingForce) {
+			case 'default':
+				return undefined;
+			case 'gentle':
+				return {
+					decay: 0.95,
+					dropoffScaleDays: 90,
+					offsetDays: 7,
+				};
+			case 'forceful':
+				return {
+					decay: 0.7,
+					dropoffScaleDays: 180,
+					offsetDays: 14,
+				};
```

Comment:
These values seem a bit surprising (e.g. why a bigger offset for forceful?), but I haven’t looked into it much and I assume you have.

Reply:
the bigger offset means that anything in the last 14 days won't get down-rated at all; it's therefore pushing recent content up the list more forcefully than dropping the score from 7 days.  It does behave in the intended way when testing though it's all a bit hard to get one's head around 😵‍💫